src/_P144_Vindriktning.ino

```diff
@@ -0,0 +1,376 @@
+      Device[deviceCount].InverseLogicOption = false;                            // Allow to invert the boolean state (e.g. a switch)
+      Device[deviceCount].FormulaOption      = true;                             // Allow to enter a formula to convert values during read. (not possible with Custom enabled)
+      Device[deviceCount].Custom             = false;
+      Device[deviceCount].SendDataOption     = true;                             // Allow to send data to a controller.
+      Device[deviceCount].GlobalSyncOption   = true;                             // No longer used. Was used for ESPeasy values sync between nodes
+      Device[deviceCount].TimerOption        = true;                            // Allow to set the "Interval" timer for the plugin.
+      Device[deviceCount].TimerOptional      = false;                            // When taskdevice timer is not set and not optional, use default "Interval" delay (Settings.Delay)
+      Device[deviceCount].DecimalsOnly       = false;                             // Allow to set the number of decimals (otherwise treated a 0 decimals)
+      break;
+    }
+
+    case PLUGIN_GET_DEVICENAME:
+    {
+      // return the device name
+      string = F(PLUGIN_NAME_144);
+      break;
+    }
+
+    case PLUGIN_GET_DEVICEVALUENAMES:
+    {
+      // called when the user opens the module configuration page
+      // it allows to add a new row for each output variable of the plugin
+      // For plugins able to choose output types, see P026_Sysinfo.ino.
+      strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[0], PSTR(PLUGIN_VALUENAME1_144));
+      break;
+    }
+
+    case PLUGIN_WEBFORM_SHOW_SERIAL_PARAMS:
+    {
+      // Called to show optinal extra UART parameters in the web interface (only called for SERIAL devices)
+      //addFormNumericBox(F("Baudrate"), P144_BAUDRATE_LABEL, P144_BAUDRATE, 2400, 115200);
+      //addUnit(F("baud"));
+      break;
+    }
+
+    case PLUGIN_GET_DEVICEGPIONAMES:
+    {
+      serialHelper_getGpioNames(event, false, true); // TX optional
+      break;
+    }
+
+    case PLUGIN_WEBFORM_SHOW_CONFIG:
+    {
+      // Called to show non default pin assignment or addresses like for plugins using serial or 1-Wire
+      string += serialHelper_getSerialTypeLabel(event);
+      success = true;
+      break;
+    }
+
+    case PLUGIN_GET_DEVICEVALUECOUNT:
+    {
+      // This is only called when Device[deviceCount].OutputDataType is not Output_Data_type_t::Default
+      // The position in the config parameters used in this example is PCONFIG(Pxxx_OUTPUT_TYPE_INDEX)
+      // Must match the one used in case PLUGIN_GET_DEVICEVTYPE  (best to use a define for it)
+      // see P026_Sysinfo.ino for more examples.
+      break;
+    }
+
+    case PLUGIN_GET_DEVICEVTYPE:
+    {
+      // This is only called when Device[deviceCount].OutputDataType is not Output_Data_type_t::Default
```

Comment:
Unused `switch` cases in the code that don't set `success = true;` should better be removed, to avoid visual 'clutter' in the source.